app/komponenter/filopplastingfelt/FilopplastningFelt.tsx

```diff
@@ -0,0 +1,64 @@
+import { DownloadIcon } from '@navikt/aksel-icons';
+import { BodyShort, Table } from '@navikt/ds-react';
+import { useState } from 'react';
+import Dropzone from 'react-dropzone';
+
+import { formaterDato } from '~/utils/formaterDato';
+
+import css from './filopplastningfelt.module.css';
+
+const FilopplastningFelt = () => {
+  const [filer, settFiler] = useState<File[]>([]);
+
+  const håndterLastOppFil = (nyeFiler: File[]) => {
+    settFiler(gamleFiler => [...gamleFiler, ...nyeFiler]);
+  };
+
+  return (
+    <>
+      <Dropzone onDrop={acceptedFiles => håndterLastOppFil(acceptedFiles)}>
+        {({ getRootProps, getInputProps }) => (
+          <section
+            className={`${css.dokumentasjonKonteiner}`}
+            {...getRootProps()}
+          >
+            <input {...getInputProps()} />
+            <section>
+              <DownloadIcon title="nedlastningsikon" />
+              <BodyShort>Last opp dokumentasjon</BodyShort>
+            </section>
```

Comment:
Hvis du ikke pakker disse to inn i noe så blir de plassert pent av flex som er definert på section over. Legg på gap og padding 1rem så blir det likt som du hadde 😁

Reply:
```suggestion
              <DownloadIcon title="nedlastningsikon" />
              <BodyShort>Last opp dokumentasjon</BodyShort>
```